docs/docs/server/openapi.yaml

```diff
@@ -96,6 +124,32 @@ paths:
         500:
           $ref: "#/components/responses/InternalServerError"
 
+  /conversations/{conversationId}/messages/{messageId}/comment:
```

Comment:
Weird that this endpoint is showing up in the diff even though it's already on `main` :/